SDK/AppCenter/Microsoft.AppCenter.Windows.Shared/Channel/Channel.cs

```diff
@@ -287,65 +311,69 @@ private void Suspend(State state, bool deleteLogs, Exception exception)
                 {
                     foreach (var log in unsentLogs)
                     {
+                        AppCenterLog.Debug(AppCenterLog.LogTag, $"Invoke FailedToSendLog event for channel '{Name}'");
                         FailedToSendLog?.Invoke(this, new FailedToSendLogEventArgs(log, exception));
                     }
                 }
                 if (deleteLogs)
                 {
-                    try
-                    {
-                        _ingestion.Close();
```

Comment:
So we will no longer close ingestion when suspending the channel? This makes sense, but what about shutting it down when the channel group is shut down?